tests/flow/test_temporal.py

```
@@ -0,0 +1,482 @@
+    def test_localdatetime_from_string(self):
+        test_cases = [
+                #('2015202T21',  '2015-07-21T21:00'),
+                #('2015T214032',  '2015-01-01T21:40:32'),
+                #('2015-W30T2140',  '2015-07-20T21:40'),
+                #('20150721T21:40',  '2015-07-21T21:40'),
+                #('2015-202T21:40:32',  '2015-07-21T21:40:32'),
+                #('2015-W30-2T214032.142',  '2015-07-21T21:40:32.142'),
+                #('2015-07-21T21:40:32.142',  '2015-07-21T21:40:32.142')
+        ]
+
+        for idx, (str_input, expected) in enumerate(test_cases):
```

Ruff (0.11.9)

Loop control variable `idx` not used within loop body

Rename unused `idx` to `_idx`

(B007)